services/venice-server/src/main/java/com/linkedin/venice/listener/StatsHandler.java

```diff
@@ -194,10 +199,10 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
        * multiple times for a single request
        */
       if (!serverStatsContext.isStatCallBackExecuted()) {
-        ServerHttpRequestStats serverHttpRequestStats =
-            serverStatsContext.getCurrentStats().getStoreStats(serverStatsContext.getStoreName());
+        ServerHttpRequestStats serverHttpRequestStats = serverStatsContext.getStoreName() == null
```

Comment:
Can `serverStatsContext.getStoreName()` be null?

Reply:
I think so since prior to the code change we do the null check within serverStatsContext. I think it could be null if the request somehow fail super early before the storeName is set?